.github/workflows/integration_test.yml

```diff
@@ -0,0 +1,39 @@
+# TODO: Support running all integration tests when it's possible to 
+# run self-hosted runners in parallel.
+name: Integration Test
+
+on:
+  pull_request:
+
+jobs:
+  changed_packages:
+    runs-on: [self-hosted, linux]
+    if: ${{ github.repository_owner == 'flutter-tizen' && github.event_name == 'pull_request' }}
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: Install flutter-tizen
+        run: |
+          git clone --depth=1 https://github.com/flutter-tizen/flutter-tizen.git
```

Comment:
how about using `actions/checkout`?
```yaml
      - uses: actions/checkout@v2
        with:
          repository: flutter-tizen/flutter-tizen
          path: flutter-tizen
```

Reply:
I'm fine either way, as long as all workflow scripts are consistent: https://github.com/flutter-tizen/plugins/blob/master/.github/workflows/build.yml#L69.

@bbrto21

Reply:
it seems nice